docker-compose.yml

```diff
@@ -1,10 +1,6 @@
 services:
-  # ======================================
-  # Conductor 
-  # --------------------------------------
-  # Orchestrates setup with health checks, data population and logging
-  # --------------------------------------
   conductor:
+    profiles: ["platform", "songDev", "scoreDev", "maestroDev", "arrangerDev", "stageDev"]
```

Comment:
not acquainted with this "profiles" property. will have to do some reading on it to provide informed feedback, but it seems redundant with its namesake env variable, so there must be a means to do away with one of them